spec/unit/face/config_spec.rb

```diff
@@ -61,7 +61,8 @@ def render(action, result)
     expect(render(:print, result)).to eq("file\n")
   end
 
-  it "prints the section and environment, and not a warning,  when a section is given" do
+  it "prints the section and environment, and not a warning, when a section is given and verbose is set" do
+    Puppet::Util::Log.expects(:sendlevel?).returns(true)
```

Comment:
Could we try using `Puppet[:log_level] = 'verbose'` instead of expecting on `sendlevel?`